Jenkinsfile

```diff
@@ -210,7 +214,7 @@ podTemplate(name: podName,
                             packagepipelineUtils.setStageEnvVars(currentStage)
 
                             // Return a map (messageFields) of our message topic, properties, and content
-                            messageFields = packagepipelineUtils.setMessageFields("package.running")
+                            messageFields = packagepipelineUtils.setMessageFields("package.running", artifact)
 
                             // Send message org.centos.prod.ci.pipeline.allpackages.package.running on fedmsg
```

Comment:
tedious but updating the comments to reflect new field too would be cool.

We will need to update the docs quite a bit soon anyways though so we can just do that with docs update doesn't matter to me